heft-plugins/heft-jest-plugin/config/rig.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Honestly I think I was trying to avoid the cyclic dependency in order to test itself using the built plugin, but that doesn't make much sense since it just means that it misses out on all the other features the rig brings with it. May as well test using the rig version of the plugin, now that things are largely stable in that area.